Khan/wonder-blocks
# Add new Banner component to Wonder Blocks

## Summary:
This is the base implementation of the Banner component. It
is mostly taken from the existing implementation of Banner
in mobile Wonder Blocks.

Updates to come in future PRs:
- 8px of spacing around floating banners
- Updating the icon to be an Icon with IconAsset instead of an img
- Changing the buttons to be small (not adding this in the current PR so that I can keep my branches consistent)
- Comprehensive accessibility features and testing
  - This includes role and aria-live

**Updated Storybook publish:**
https://5e1bf4b385e3fb0020b7073c-omnvzidaqs.chromatic.com/?path=/story/banner--default

Old Storybook publishes:
https://5e1bf4b385e3fb0020b7073c-fdyhsmnhub.chromatic.com/Ï
https://5e1bf4b385e3fb0020b7073c-meqidgklni.chromatic.com/
https://5e1bf4b385e3fb0020b7073c-aclcwsrtdq.chromatic.com/
https://5e1bf4b385e3fb0020b7073c-eytllurxkb.chromatic.com/
https://5e1bf4b385e3fb0020b7073c-gnzcmyyvts.chromatic.com/

Issue: https://khanacademy.atlassian.net/browse/WB-1384

## Test plan:
Storybook
- Open Banner in storybook.
- Check that all the controls in the control panel work.
- Check that all the descriptions make sense.
- Check that the banner looks correct using the Storybook
  ruler and outline tools.

Unit tests
- Run `yarn jest packages/wonder-blocks-banner/src/components/__tests__/banner.test.js`
